src/registry.json

```diff
@@ -2596,6 +2596,42 @@
         }
       }
     },
+    "@nufi/cardano-metamask-snap ": {
```

Comment:
The curve that this Snap needs is not in production yet. We can either approve it as hidden for now or we can wait with approval until it is in production. We don't wanna surface non-functional Snaps in the Directory.

Reply:
I remove the category field to potentially approve as hidden. I will check with nu.fi team to see what they prefer. 

Do you know what is the expected time of arrival of the feature they need in production ?